packages/odf/components/actions/csv-actions.ts

```diff
@@ -113,3 +146,20 @@ const ConfigurePerformanceStorageSystem = (
     },
   };
 };
+
+const CapacityAutoscalingAction = (
+  launchModal: LaunchModal,
+  storageCluster: StorageClusterKind
+): Action => {
+  return {
+    id: 'capacity-autoscaling-action',
+    label: 'Smart capacity scaling',
+    insertAfter: 'configure-performance-storage-system',
+    cta: () => {
+      launchModal(CapacityAutoscalingActionModal as any, {
```

Comment:
```suggestion
      launchModal(CapacityAutoscalingActionModal, {
```
add `closeModal: null` as a prop

Reply:
In fact `any` was masquerading a property mismatch due to an unnecessary level of indirection (CapacityAutoscalingActionModal): removed `CapacityAutoscalingActionModal` and properly set type in `CapacityAutoscalingModal`